visidata/loaders/_pandas.py

```diff
@@ -90,9 +95,9 @@ def sort(self):
         'Sort rows according to the current self._ordering.'
         by_cols = []
         ascending = []
-        for cols, reverse in self._ordering[::-1]:
-            by_cols += [col.name for col in cols]
-            ascending += [not reverse] * len(cols)
+        for col, reverse in self._ordering[::-1]:
```

Comment:
@saulpw, can `self._ordering[::]` contain multiple columns? Probably when you have multiple key columns, and do `g[`?

Reply:
Yes; each item in `_ordering` should be a tuple of `(list(columns), reverse_bool)`.

Reply:
Judging from @ajkerrigan's linked issue report: https://github.com/saulpw/visidata/issues/495

It has not been a `list(columns)`, so we should look into that!

Reply:
I did do some tests locally with `z[`/`z]` to make sure this behaved when sorting by multiple columns. Because in those cases, `self._ordering` looks like:

```[(<Column 1>, True), (<Column 2>, False)]```

Still definitely possible that I missed a piece of behavior, but as far as I can tell sorting works with single or multiple columns.

Reply:
My apologies, I changed it a few months ago in 9da019ca4, to be `(column, reverse_bool)`.  So that is the issue, that the "interface" for `_ordering` changed and the pandas loader did not change with it.  I wonder if `_ordering` should lose its private `_` and become a documented interface?  Other loaders/plugins that override provide custom sort functionality would need access to it as well, so we should probably formalize it.

Reply:
I can also confirm that it does work!

Reply:
Agreed about formalizing `_ordering`, and properly documenting it..